compiler/tests/fixtures/features/return_msg_sig.fe

```diff
@@ -0,0 +1,5 @@
+contract Foo:
+    pub def bar() -> bytes[32]:
+        msgSig: bytes[32]
```

Comment:
The idiomatic way to write Fe code is with `snake_case`. Please change `msgSig` to `msg_sig`.